andela/titan-dev-ah-frontend
# 163518646  View and Edit Profile

#### What does this PR do?
It lets a user view and edits their profile
#### Description of Task to be completed?
-  Create `Profile` components with React
-  Create `EditProfile` components with React
-  Create `Settings` components with React
- Integrate the components with redux for state management
- write unit tests for all elements used by those components 
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-view-edit-profile-#23163518646`
- clone the repo and check out the following branch `ft-change-password-#23165713051` from our backend [repo](https://github.com/andela/Titan-Devs/pull/54) 
-  Refer to `.sample.env` to update your env file
- run `yarn start` 
-  Navigate `localhost:{port}/profiles/:{username}` for viewing a profile
 - Navigate `localhost:{port}/profiles/:{username}/edit` for editing a profile
 - Navigate `localhost:{port}/settings` for user settings
- run  `yarn test` to  run the unit tests
#### Any background context you want to provide?
The PR depends on the logged in user.
#### What are the relevant pivotal tracker stories?
#163518642 
#### Screenshots (if appropriate):
View page
<img width="1284" alt="Screen Shot 2019-04-14 at 13 40 07" src="https://user-images.githubusercontent.com/25322304/56092334-3f92ee80-5ebb-11e9-8d55-2a956ba5b4c1.png">
Edit page
<img width="1050" alt="Screen Shot 2019-04-14 at 13 40 30" src="https://user-images.githubusercontent.com/25322304/56092337-46216600-5ebb-11e9-850b-ff0f4b5b6bd3.png">
